recipe/meta.yaml

```diff
@@ -1,28 +1,35 @@
 {% set name = "opentelemetry-semantic-conventions" %}
-{% set alt_name = name.replace('-','_') %}
-{% set version = "0.44b0" %}
-{% set sha256 = "2e997cb28cd4ca81a25a9a43365f593d0c2b76be0685015349a89abdf1aa4ffa" %}
+{% set version = "0.47b0" %}
```

Comment:
Today 0.48b0 has been released https://pypi.org/project/opentelemetry-semantic-conventions/#history. Or are we fine with 0.47b0?

Reply:
Hmm, `0.48b0` requires `opentelemetry-api == 1.27.0`

Reply:
Yeah we're going to be one version behind for this one just because of the delay, all these libraries are linked

Reply:
From the perspective of Snowflake, I like the plan of moving forward with 0.47b0 opposed to delaying if thats worth anything.